src/components/LinearProgressBar/LinearProgressBar.tsx

```diff
@@ -32,3 +32,12 @@ export const LinearProgressBar: React.SFC<LinearProgressBarProps> = (props: Line
   )
 }
 
+LinearProgressBar.propTypes = {
+  value: PropTypes.number,
```

Comment:
can spread 
```
...CoreLinearProgressBar.propTypes,

```

Reply:
CoreLinearProgressBar doesn't have propTypes, should we add it as well?